src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceACL.scala

```diff
@@ -10,20 +10,18 @@ case class WorkspaceACLUpdate(email: String, accessLevel: WorkspaceAccessLevel)
 
 object WorkspaceAccessLevels {
   sealed trait WorkspaceAccessLevel extends RawlsEnumeration[WorkspaceAccessLevel] with Ordered[WorkspaceAccessLevel] {
-    val all = Seq(UnknownUser, NoAccess, Read, Write, Owner)
-
     def compare(that: WorkspaceAccessLevel) = { all.indexOf(this).compare(all.indexOf(that)) }
 
     override def toString = WorkspaceAccessLevels.toString(this)
     override def withName(name: String) = WorkspaceAccessLevels.withName(name)
   }
 
-  case object UnknownUser extends WorkspaceAccessLevel
```

Comment:
UnknownUser does not make sense as a workspace access level and I don't see it used anywhere.